Makefile

```diff
@@ -15,11 +15,24 @@ check:
 check-dummy:
 	BUILD_DUMMY_WASM_BINARY= cargo check
 
+# Pseudo private target is invoked by public targets for different chains
+--try-runtime:
+	RUST_LOG=runtime=trace,try-runtime::cli=trace,executor=trace \
+		cargo run \
+		--bin=zeitgeist \
+		--features=parachain,try-runtime \
+		try-runtime \
+		--execution=Native \
+		--chain=${TRYRUNTIME_CHAIN} \
+		on-runtime-upgrade \
+		live \
+		--uri=${TRYRUNTIME_URL}
+
 try-runtime-upgrade-battery-station:
-	cargo run --release --bin=zeitgeist --features=parachain,try-runtime try-runtime on-runtime-upgrade live --uri wss://bsr.zeitgeist.pm:443
+	@$(MAKE) TRYRUNTIME_CHAIN="battery_station_staging" TRYRUNTIME_URL="wss://bsr.zeitgeist.pm:443" -- --try-runtime
 
 try-runtime-upgrade-zeitgeist:
-	cargo run --release --bin=zeitgeist --features=parachain,try-runtime try-runtime on-runtime-upgrade live --uri wss://zeitgeist-rpc.dwellir.com
+	@$(MAKE) TRYRUNTIME_CHAIN="zeitgeist_staging" TRYRUNTIME_URL="wss://zeitgeist-rpc.dwellir.com:443" -- --try-runtime
```

Comment:
Can you briefly explain, why and what the effect of using `staging` chains means? So I think staging is a pre-release chain specification?